dracut/anaconda-diskroot

```diff
@@ -64,7 +64,7 @@ fi
 # retrigger this script, and we'll mount the disk as normal.
 if str_starts "$kickstart" "cdrom" && [ ! -e /tmp/ks.cfg.done ]; then
     if dev_is_cdrom "$dev"; then
-        > /tmp/anaconda-on-cdrom
+        true > /tmp/anaconda-on-cdrom
```

Comment:
Couldn't we rather use `touch` than this form?

Reply:
Good point. I was so hung up on "fixing the issues" that I missed the obvious solutions :-/

Reply:
No `touch` in initramfs, I'm afraid.

Reply:
lol, ok